src/PerformanceTests/Common/Program.cs

```diff
@@ -56,12 +56,12 @@ static int Main()
 
                     if (Environment.UserInteractive) Console.Title = PermutationParser.ToFriendlyString(permutation);
 
-                    var runnerTypes = AssemblyScanner.GetAllTypes<BaseRunner>();
+                    var runnerTypes = AssemblyScanner.GetAllTypes<BaseRunner>().ToArray();
 
-                    foreach(var t in runnerTypes) Log.Error(t.FullName);
+                    foreach (var t in runnerTypes) Log.DebugFormat("Found runner: {0}", t.FullName);
```

Comment:
In core we don't use the shortcut syntax. But this is your call.

The same comment applies for all the if's down below.